--- SConstruct.orig	2014-07-03 17:38:05.000000000 +1200
+++ SConstruct	2014-07-03 17:37:27.000000000 +1200
@@ -140,7 +140,7 @@
 # installation
 opts.Add(('PREFIX', 'Installation prefix', dirs['prefix']))
 opts.Add(ListVariable('SKIPSTUBS', 'A list of stubs that will not be built', 'none', stubs))
-opts.Add(ListVariable('SKIPPLUGINS', 'A list of plug-ins that will not be built', 'none', plugins))
+opts.Add(ListVariable('SKIPPLUGINS', 'A list of plug-ins that will not be built', 'none', plugin_libs + plugins))
 opts.Add(ListVariable('SKIPUTILS', 'A list of utilities that will not be built', 'none', utils))
 opts.Add(ListVariable('SKIPMISC', 'A list of plug-ins that will not be built', 'none', misc))
 opts.Add(ListVariable('SKIPDOC', 'A list of doc files that will not be built/installed', 'none', doc))
@@ -151,6 +151,8 @@
 opts.Add(('TOOLSET', 'A comma-separated list of specific tools used for building instead of the default', None))
 opts.Add(BoolVariable('MSTOOLKIT', 'Use Microsoft Visual C++ Toolkit', 'no'))
 opts.Add(BoolVariable('CHMDOCS', 'Build CHM documentation, requires hhc.exe', hhc))
+opts.Add(('CC', 'Override C compiler', None))
+opts.Add(('CXX', 'Override C++ compiler', None))
 opts.Add(PathVariable('APPEND_CPPPATH', 'Additional paths to search for include files', None))
 opts.Add(PathVariable('APPEND_LIBPATH', 'Additional paths to search for libraries', None))
 opts.Add(('APPEND_CCFLAGS', 'Additional C/C++ compiler flags'))
